app/packages/rust_sdk/lib/acter_flutter_sdk.dart

```diff
@@ -372,20 +372,20 @@ class ActerSdk {
       // android api 30 is working here
       return DynamicLibrary.open(libName);
     } catch (e1) {
-      debugPrint('DynamicLibrary.open by lib name failed: $e1');
+      _log.severe('DynamicLibrary.open by lib name failed: $e1');
```

Comment:
nope. the idea of `_log` was to go _through_ the API, but these things here are failing so we can't log through `_log`. These need to work regardless.

```suggestion
      debugPrint('DynamicLibrary.open by lib name failed: $e1');
```

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1495/commits/a6c2799abeb8622694e4e2c9c9ef14dd9cea0007